web-app/js/portal/search/FacetedSearchResultsDataView.js

```diff
@@ -221,12 +236,20 @@ Portal.search.FacetedSearchResultsDataView = Ext.extend(Ext.DataView, {
     _getPlatformAsHtml: function(template, platforms) {
 
         var label = this._buildLabel("fa-tags", OpenLayers.i18n('searchPlatformText'));
-        var broaderPlatforms = this._getBroaderTerms(platforms, 1,'platformCategories');
+
+        var termType = ''
+        if (Portal.app.appConfig.geonetwork.version === 3) {
+            termType = 'platformCategories'
+        } else {
+            termType = 'Platform'
+        }
+
```

Comment:
Could we abstract these if(version === x) { something = something } type bits into a CatalogueTermLookup class or something?  It could have getPlatformCategoryTerm(version), getOrganisationTerm(version), getParameterCategoryTerm(version) methods